symfony/messenger/4.1/manifest.json

```diff
@@ -3,7 +3,23 @@
         "config/": "%CONFIG_DIR%/"
     },
     "env": {
-        "#1": "MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages"
+        "#1": "MESSENGER_TRANSPORT_DSN=amqp://symfony:password@localhost:5672/%2f/messages"
+    },
+    "docker-compose": {
+        "services": {
+            "rabbitmq": {
+                "image": "rabbitmq:3.7.5",
+                "environment": [
+                    "# You should definitely change the password in production",
+                    "- RABBITMQ_DEFAULT_USER=symfony",
+                    "- RABBITMQ_DEFAULT_PASS=password"
```

Comment:
consider adding `RABBITMQ_NODENAME` also.

https://docs.docker.com/samples/library/rabbitmq/
> One of the important things to note about RabbitMQ is that it stores data based on what it calls the “Node Name”, which defaults to the hostname. What this means for usage in Docker is that we should specify -h/--hostname explicitly for each daemon so that we don’t get a random hostname and can keep track of our data